.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Build test
+'on':
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Is it possible to also run this on other distros?  Testing on a variety would be quite valuable.

Reply:
I think Ubuntu is the only (freely) available distro. You can pick older versions though.
See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

Also adding custom runners should be possible, but I don't know how that works.

Reply:
It is possible to test with multiple distribution using docker. I can try to help with that after this is merged.

Reply:
Ok.  It's a pity it's not easier to test other things, but something is better than nothing.  @Biswa96 that would be most appreciated i fyou could extend the coverage in future.